samples/task-dependencies/build.gradle.kts

```diff
@@ -1,21 +1,47 @@
+/**
+ * An example Gradle task.
+ */
+open class GreetingTask : DefaultTask() {
```

Comment:
Has the `all-open` plugin not been applied to build scripts yet? Is there still a plan to do so?

Reply:
No, its not.
See https://github.com/gradle/kotlin-dsl/issues/390